sequences/allied-infantry.yaml

```diff
@@ -288,6 +288,60 @@ spy:
 		Tick: 80
 	icon: spyicon
 
+seal: # TODO: seala on SNOW
+	stand:
+		Facings: 8
+	run:
+		Start: 8
+		Length: 6
+		Facings: 8
+		Tick: 120
+	idle1:
+		Start: 56
+		Length: 15
+		Tick: 100
+	idle2:
+		Start: 71
+		Length: 15
+		Tick: 100
+	prone-run:
+		Start: 86
+		Length: 6
+		Facings: 8
+		Tick: 100
+	die1:
+		Start: 134
+		Length: 14
+	die2:
+		Start: 149
+		Length: 14
+	die4: nukedie
+		Length: *
+		Tick: 50
+		# Problem: Palette = effect
```

Comment:
Shouldn't be a problem anymore: https://github.com/OpenRA/ra2/blob/master/rules/defaults.yaml#L278